clock.rb

```diff
@@ -0,0 +1,14 @@
+# rubocop:disable Style/MixinUsage
+require "clockwork"
+require File.expand_path("../config/environment", __FILE__)
+include Clockwork
+
```

Comment:
Hi @edwinthinks do you have schedule task list in Heroku, to put in this file?
Or will we be other PR to create this?

Reply:
The only scheduled task is the one in the file already :)